From c3bcb63ba8715e4de087892d2a3f6216b7142f39 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 12 Nov 2013 19:57:29 +0000
Subject: [PATCH] builder: Clarify architecture section of the documentation.

I have checked and cross architecture builds work fine:

http://rwmj.wordpress.com/2013/11/09/virt-builder-running-on-arm/#content
(cherry picked from commit 544d6b25e345e2317b1f2e92684af6c08e1c58ba)
---
 builder/virt-builder.pod | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/builder/virt-builder.pod b/builder/virt-builder.pod
index bad59c1..045adfa 100644
--- a/builder/virt-builder.pod
+++ b/builder/virt-builder.pod
@@ -1322,16 +1322,15 @@ highly recommended that you always create signed index and templates.
 
 =head2 ARCHITECTURE
 
-Virt-builder can, in theory, build a guest for any architecture no
-matter what the host architecture is.  For example a ppc64 guest on an
-x86-64 host.
+Virt-builder can build a guest for any architecture no matter what the
+host architecture is.  For example an x86-64 guest on an ARM host.
 
 However certain options may not work correctly, specifically options
 that require running commands in the guest during the build process:
 I<--install>, I<--run>, I<--run-command>.  You may need to replace
 these with their firstboot-equivalents.
 
-An x86-64 host building 32 bit x86 guests should work without any
+An x86-64 host building 32 bit i686 guests should work without any
 special steps.
 
 =head2 SECURITY
-- 
1.8.3.1

